src/main/java/io/curity/identityserver/plugin/webauthn/WebAuthnAuthenticationSession.java

```diff
@@ -0,0 +1,93 @@
+/*
+ *  Copyright 2020 Curity AB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.curity.identityserver.plugin.webauthn;
+
+import com.google.common.collect.ImmutableSet;
+import se.curity.identityserver.sdk.Nullable;
+import se.curity.identityserver.sdk.attribute.Attribute;
+import se.curity.identityserver.sdk.service.ExceptionFactory;
+import se.curity.identityserver.sdk.service.SessionManager;
+
+import java.util.Set;
+import java.util.UUID;
+
+import static se.curity.identityserver.sdk.errors.ErrorCode.INVALID_INPUT;
+
+/**
+ * Represents an immutable pair with the WebAuthn authentication session ID and the username that was used to start it.
+ */
+public class WebAuthnAuthenticationSession
+{
+    static final String WEBAUTHN_USERNAME_KEY = "webauthn_username";
+    static final String WEBAUTHN_SESSION_ID_KEY = "webauthn_session_id";
+    static final Set<String> webAuthnSessionKeys = ImmutableSet.of(WEBAUTHN_SESSION_ID_KEY, WEBAUTHN_USERNAME_KEY);
```

Comment:
Any reason to expose this instead of making it private?

Reply:
It could be private. Will make the change.